authentikos/README.md

```diff
@@ -53,3 +54,4 @@ The following is a list of supported options for `authentikos`:
 - 0.0.3: add new `TimeToUnix`, `UnixToTime`, and `Parse` template variable and change method signature for math template variables from `(a, b time.Duration) time.Duration` to `(a, b int64) int64`.
 - 0.0.4: add `--key` option for specifying the name of the data key in the created Kubernetes secret.
 - 0.0.5: use [Sprig](http://masterminds.github.io/sprig/) as the library for template functions.
+- 0.0.6: add `--force-refresh` option for forcing a token refresh even if not yet expired. If this option is omitted or false, credentials will only refresh after expiration.
```

Comment:
Should this be the default behavior?

Reply:
I prefer to be backwards compatible. Also, this is consistent with the default behavior of `gcloud config config-helper` and the oauth2 Go client library.

Reply:
Only because I like default booleans to be `false`, I renamed this to `--reuse` and made the default `false`, so force refreshing is now the default behavior.